docs/code_tour.md

```diff
@@ -0,0 +1,22 @@
+# Code Tour 
+This document is intended to provide a development guideline for new and existing contributors to the project.
+Keep in mind this application undergoes constant development so be sure to check back occasionally as best practices are sure to evolve.
+
+## Linting
+This project uses **ESLint** and **Sass Lint**.  Both can be run with the command `gulp vet`.
+
+## Style and Convention
+For additional information regarding coding style and convention employed in this project checkout:
+* [Coding Style and Standards](https://github.com/ManageIQ/manageiq/issues/8781)
+* [Internationalization Guidelines](i18n.md) 
```

Comment:
Just noticed that this link is broken, I believe it is supposed to link to MangeIQ/Guides?

https://github.com/ManageIQ/guides/blob/master/i18n.md